config/HELICSConfig.cmake.in

```diff
@@ -16,14 +16,12 @@
 set(PN HELICS)
 set(${PN}_INCLUDE_DIR "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_INCLUDEDIR@")
 
-set(C_SHARED_BUILT @BUILT_C_SHARED_LIB@)
-if (C_SHARED_BUILT )
+if (@BUILD_C_SHARED_LIB@)
```

Comment:
Does this work,  I thought I remember trying it before and ending up with some things like if (ON) which cmake wasn't really interpreting correctly

Reply:
It works with CMake 3.5.2 and 3.13.4 on LC, and the docs going back to 3.0 say if(<constant>) is supported.